help/access/databases.md

```diff
@@ -305,194 +313,241 @@ supported. You can add an input/output of type `list<string>`.
+    AND @shipVia(shipVia)
+    AND ((freight < 1000) OR NOT @freightLess1000)
+    AND (shipCountry = @shipCountry)
+    AND @shipCity(shipCity)
+    AND @orderDate(orderDate)
+    AND (requiredDate >= @requiredDate)
+```
 
-To share query results as URL, first, run the query. Once the query is executed, copy the URL from the address bar.
-Query parameters are encoded within the URL. Follow the link to _open_ the dataframe in Datagrok. To access the query
-results from the link provided, users must have permissions to execute this query.
+</TabItem>
+<TabItem value="parameter-dialog" label="Parameter dialog">
+
+![Parametrized queries](parameterized-queries.png)
+
+</TabItem>
+</Tabs>
+
+:::note developers
+
+You can [_call_ a query](../datagrok/functions/function-call.md) across multiple Datagrok plugins instead of copy-pasting the same query every time. To run a query programmatically, see this [code snippet](https://public.datagrok.ai/js/samples/data-access/parameterized-query). Learn how to [expose the parameter dialog to end-users as an application](../develop/how-to/create-package.md).<!--Mention?: when the cartdridge is not deployed on that particular database, the query returns an error-->
+
+:::
+
+After running a query, you typically get back a dataframe. You can perform various operations on this dataframe, such as data cleansing, transformation, [and more](../visualize/viewers/grid.md). All of these actions get recorded as macros, and will be replayed when the query is re-executed. Once you have the data set up to your satisfaction, you can add viewers and create a visualization.  
+
+To view the queries you've created or those shared with you, you can use the **Queries Gallery** (**Data** > **Queries**). This gallery provides a convenient interface to quickly access and manage queries. You can search queries by their name or tag and use the **Context Panel** to view information and actions for the selected query.
+
+## Creating views for query results
+
+You can persist the view of the query output and expose it to end users as a dashboard. For parameterized queries, these dashboards can be dynamic, allowing dashboard users to change the query parameters right inside the dashboard to view different datasets and interact with data in real-time.
+
+To save the query output as a dynamic dashboard, do the following:
+
+1. Run a query to open a dataframe and create a desired view of the query output.
+1. On the **Sidebar**, click **Projects** > **Upload**.
+1. In the **Upload project** dialog, enter a name and description (optional) in the fields provided.
+1. Select how to store data:
+    * Save the data as a static snapshot.
+    * Store the data as a generation script by toggling the **Data sync** control. The query re-executes each time the project is opened. To learn more about dynamic data updates in projects, see [Dynamic data](../datagrok/project.md/#dynamic-data).
+1. Click **OK** to upload the project.
```

Comment:
Need to mention how other users are going to get it. You need to share it (this is important, otherwise they won't see it), and then they will get an email with the link, also the dashboard will be available under "projects"